lib/entities/accepted_terms_and_conditions.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:concordium_wallet/services/wallet_proxy/model.dart';
+import 'package:hive_flutter/hive_flutter.dart';
+
+part 'accepted_terms_and_conditions.g.dart';
+
+/// Version of the Terms & Conditions accepted by the user.
+@HiveType(typeId: 1)
+class AcceptedTermsAndConditions {
+  static const table = "accepted_terms_and_conditions";
+
+  @HiveField(1)
```

Comment:
Looking at the documentation `@HiveField` should be `0`-indexed, so start with `0`.